src/main/resources/jenkins/plugins/hipchat/workflow/HipChatStep/config.jelly

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="message" title="Message">
+        <f:textbox/>
+    </f:entry>
+    <f:entry field="color" title="Color">
```

Comment:
if you change the type of the `color` field to `jenkins.plugins.hipchat.model.Color`, then something like
```
<f:entry title="${%Color}" field="color">
  <f:enum>${it}</f:enum>
</f:entry>
```
should work.